manifest-okd-c10s.yaml

```diff
@@ -0,0 +1,115 @@
+# Manifest for OKD node based on CentOS Stream CoreOS 1à
+# Note: this manifest is temporary; in the future, OKD components will be layered instead.
+
+rojig:
+  license: MIT
+  name: scos
+  summary: OKD 4.19
+
+variables:
+  osversion: "c10s"
+
+# Include manifests common to all RHEL and CentOS Stream versions and manifest
+# common to RHEL 10 & C10S variants
+include:
+  - manifest-c10s.yaml
+  - packages-openshift.yaml
+
+# Additional repos we need for OKD components
+repos:
+  # For containernetworking-plugins for cri-o
+  - rhel-9.6-appstream
```

Comment:
temporary?

Reply:
yes, it's temporary until we have this published in the RHAOS repo or cri-o stops depending on it in the RHEL 10 if that's possible.